### PR TITLE
add faq question about ClusterAutoscaler and MachineAutoscaler

### PR DESCRIPTION
**Pull request by elmiko**
this change adds a question describing the differences between these two
resources and when you would use them. also gives links to examples.

/kind documentation

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -146,6 +147,13 @@ Any change to a MachineDeployment will result in an immediate removal and replac
 
 # Cluster Autoscaler
 
+## What is the difference between the ClusterAutoscaler and MachineAutoscaler resources?
+The ClusterAutoscaler resource is used to manage the lifecycle of the Kubernetes cluster autoscaler. It can be used to specify how the cluster autoscaler should be deployed and what global system limits it should respect. You will only need to create one of this resource.
```
[the remark this answers is not in the document]
Comment (elmiko) [reply FAQ.md]: changed, thanks!

**Comment by elmiko** [comment]
updated with language change for ClusterAutoscaler

File: FAQ.md [hunk 2]
```diff
@@ -146,6 +147,13 @@ Any change to a MachineDeployment will result in an immediate removal and replac
 
 # Cluster Autoscaler
 
+## What is the difference between the ClusterAutoscaler and MachineAutoscaler resources?
```
Comment (enxebre) [inline FAQ.md:150]: we need to add a reference once we merge https://github.com/openshift/enhancements/pull/332

Comment (elmiko) [reply FAQ.md:150]: a reference to the enhancement doc?

**Pull request merged**